packages/@lwc/engine-server/src/__tests__/fixtures/getter-class-list-modify/error.txt

```diff
@@ -0,0 +1 @@
+classList.forEach is not a function
```

Comment:
This PR introduces functionality that is not present in SSRv1. It is unclear to me how the (internal) components that were failing with SSRv2 ever rendered with SSRv1, unless the codepath just wasn't hit for some reason. In any event, we'